src/db/db.js

```diff
@@ -0,0 +1,16 @@
+import Sequelize from 'sequelize';
+import dotenv from 'dotenv';
```

Comment:
config may contain env varibles for different parts of application, not only the database connection.
So the best way is to initialise it in the root module before any other imports

app.js
```
import 'dotenv/config'; //import before any other imports

import express from 'express';
import router from './routers/index.js';
import handleError from './middleware/errorHandler.js';

const app = express();

app.listen(3000);
app.use(express.json());
app.use('/api', router);
app.use('/api', handleError);
```

Reply:
Done